io/src/test/scala/sbt/internal/io/PollingWatchServiceSpec.scala

```diff
@@ -1,12 +1,13 @@
 package sbt.internal.io
 
-import sbt.io.PollingWatchService
-
+import sbt.io.{ IO, PollingWatchService }
 import scala.concurrent.duration._
 
 class PollingWatchServiceSpec
     extends SourceModificationWatchSpec(
       new PollingWatchService(500.milliseconds),
       500.milliseconds,
       3.seconds
-    )
+    ) {
+  Milli.getMilliSupportDiagnostic(IO.temporaryDirectory) map println
```

Comment:
This should really be the project dir; the temporary directory on *nix is typically on /tmp, which is tmpfs, a virtual ram-based file system. So, it's not really checking the file system at all.

Reply:
Same for the `createTempFile()`, below

Reply:
Good point..